pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1306,6 +1308,62 @@ func (og *operationGenerator) GenerateExpandVolumeFunc(
 	}, nil
 }
 
+func (og *operationGenerator) GenerateVolumeFSResizeFunc(
```

Comment:
Can the naming and comments make it clear that method will do FS resizing without unmounting (and how it relates to `resizeFileSystem` called inside `GenerateMountVolumeFunc`?

Reply:
Changed `GenerateVolumeFSResizeFunc` to `GenerateExpandVolumeFSWithoutUnmountingFunc`.